runtime/onert/backend/cpu/ops/BinaryArithmeticLayer.cc

```diff
@@ -126,9 +126,6 @@ void setAddOrSubQuant8Params(const IPortableTensor *lhs, const IPortableTensor *
   op_params.input1_offset = -lhs->data_zero_point();
   op_params.input2_offset = -rhs->data_zero_point();
   op_params.output_offset = output->data_zero_point();
-  assert((op_params.input1_offset <= 0) && (op_params.input1_offset >= -255));
-  assert((op_params.input2_offset <= 0) && (op_params.input2_offset >= -255));
-  assert((op_params.output_offset >= 0) && (op_params.output_offset <= 255));
```

Comment:
This assert is not valid for `int8`. Thus, it should be updated. However, I would like to check validity of zero_point in `OperationValidator`. I will do in another PR.